src/cli.js

```diff
@@ -128,18 +139,11 @@ if (cli.flags.quiet) {
 }
 
 if (cli.flags.syntax) {
-  optionsBase.syntax = cli.flags.syntax
+  optionsBase.syntax = resolveModule(cli.flags.syntax)
```

Comment:
But `scss` or `less` are not going to resolve to the `postcss-scss` and `postcss-less` modules, right?